### PR TITLE
bootstrap is added

### PR DESCRIPTION
**Pull request by Sushilverma002**
## Screenshots

image

### CODE REVIEW

File: database/frontend/bootstrap.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "name": "Introduction",
+        "description": "Get started with Bootstrap, the world most popular framework for building responsive, mobile-first sites, with jsDelivr and a template starter page.",
```
Comment (CBID2) [inline database/frontend/bootstrap.json:4]:
```suggestion
        "description": "Get started with Bootstrap, the world's most popular framework for building responsive mobile-first sites with jsDelivr and a template starter page.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Sushilverma002 ! :) I pointed out some typos and proofreading errors that need to be fixed. Also, the deployment has failed for some reason so I suggest waiting for @rupali-codes' review.

**Review by rupali-codes — COMMENTED** [review]
Hi @Sushilverma002 can you please go through CONTRIBUTING guidelines once?

**Review by rupali-codes — CHANGES_REQUESTED** [review]
* update your branch with `linkshub:main`
* you don't have to create a NEW subcategotry, please add this under [resources/official-docs,json](https://github.com/rupali-codes/LinksHub/blob/main/database/resources/official-docs.json)

**Pull request closed without merging**